### ISSUE TITLE
Spaces in command parameters break createcontrail

### ISSUE THREAD

**Issue by buoto** [codilime/contrail-lab#16]
If any of provided parameters contain space then `createcontrail` breaks with meaningless error message.
```
Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
The state file either has no outputs defined, or all the defined
outputs are empty. Please define an output in your configuration
with the `output` keyword and run `terraform refresh` for it to
become available. If you are using interpolation, please verify
the interpolated value is not empty. You can use the 
`terraform console` command to assist.
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```
This is very hard to spot especially when space is in password field.

**Comment by mlynekt** [comment]
It will be fixed in #14

**Comment by mlynekt** [comment]
closing as https://github.com/codilime/contrail-lab/pull/14 is merged
